modules/rules_oci/1.4.2/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+bcr_test_module:
+  module_path: "e2e/crane_as_registry"
+  matrix:
+    # TODO(#97): add windows
```

Comment:
@thesayyn could you TAL at the PR for solving this? I rebased it last night and don't see how the error is possible